tests/sqlparser_common.rs

```diff
@@ -2474,10 +2474,10 @@ fn parse_create_table() {
                constrained INT NULL CONSTRAINT pkey PRIMARY KEY NOT NULL UNIQUE CHECK (constrained > 0),
                ref INT REFERENCES othertable (a, b),\
                ref2 INT references othertable2 on delete cascade on update no action,\
-               constraint fkey foreign key (lat) references othertable3 (lat) on delete restrict,\
-               constraint fkey2 foreign key (lat) references othertable4(lat) on delete no action on update restrict, \
-               foreign key (lat) references othertable4(lat) on update set default on delete cascade, \
-               FOREIGN KEY (lng) REFERENCES othertable4 (longitude) ON UPDATE SET NULL
+               constraint fkey foreign key (lat) references othertable3 (lat) on delete restrict deferrable initially deferred,\
```

Comment:
Can you please add a new test rather than modifying an existing test so it is more clear this PR does not change existing parsing behavior?